Dockerfile

```diff
@@ -15,9 +15,9 @@ WORKDIR /go/src/github.com/prometheus/client_golang/examples/simple
 RUN CGO_ENABLED=0 GOOS=linux go build -a -tags netgo -ldflags '-w'
 
 # Final image.
-FROM prom/busybox
+FROM quay.io/prometheus/busybox:latest
```

Comment:
Looking at prometheus/busybox repo, quay seems to be current hosted registry for the busybox image.

Reply:
correct